.github/workflows/cron-nixpkgs.yml

```diff
@@ -4,6 +4,7 @@ on:
 
   schedule:
   - cron: '0 * * * *'
+  pull_request:
```

Comment:
Should this be removed?

Reply:
Yes
I should think of a better way to do this debugging things